pages/index.js

```diff
@@ -51,10 +51,10 @@ export default class IndexPage extends React.PureComponent {
             ))}
           </ul>
         </main>
+        
```

Comment:
Can you remove this change from your commit? You can ask in git_help in case you face an issue :)